settings/Mailing.setting.php

```diff
@@ -393,4 +393,18 @@
     'description' => ts('The frequency that CiviMail updates its sent mail database.'),
     'help_text' => NULL,
   ],
+  'scheduled_reminder_smarty' => [
+    'group_name' => 'Mailing Preferences',
+    'group' => 'mailing',
+    'name' => 'scheduled_reminder_smarty',
```

Comment:
If you also add this to https://github.com/civicrm/civicrm-core/blob/c88ead2a84121c618192134830310bd83f54df67/CRM/Admin/Form/Preferences/Mailing.php#L37 then it will automatically have a UI for it at admin/setting/preferences/mailing

Reply:
Done